pupil_src/launchables/eye.py

```diff
@@ -370,11 +370,28 @@ def set_scale(new_scale):
         )
 
         def set_window_size():
+            # Get current capture frame size
             f_width, f_height = g_pool.capture.frame_size
-            f_width *= 2
-            f_height *= 2
-            f_width += int(icon_bar_width * g_pool.gui.scale)
-            glfw.glfwSetWindowSize(main_window, f_width, f_height)
+            frame_scale_factor = 2
+
+            # Scale the capture frame by a constant scale factor
+            f_width *= frame_scale_factor
+            f_height *= frame_scale_factor
```

Comment:
The added comment still doesn't tell me why we do this :stuck_out_tongue: 
Is this just so that the eye windows are more clearly visible?